notebooks/Text_Extensions_for_Pandas_Overview.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>The first sentence is a bit confusing. maybe reword it to "Text Extensions for Pandas provides several Pandas extension arrays to the end user. Much of the functionality of this library is built on top of these extension arrays."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>